ios/authentication.md

```diff
@@ -1014,6 +1014,14 @@ Note: your user pool domain is something like: `domain_prefix-<env-name>.auth.<r
 ![Image]({{common_media}}/cognitoHostedUI/amazon5.png)
 5. Choose Save.
 
+##### Setting up Hosted UI Domain with Apple
```

Comment:
By "with Apple" do you mean "Sign in with Apple?" If so, is it worth it to be explicit?

Reply:
previous sections say "with Amazon", "with Google", however i have heard apple's referenced as "Apple Sign In". so i'm not sure
"Setting up Hosted UI Domain for Sign In with Apple"
"Setting up Hosted UI Domain for Apple Sign In"

Reply:
"Setting up Hosted UI Domain for Sign In with Apple"